my-app/src/context/GameContext.js

```diff
@@ -0,0 +1,7 @@
+import React, { createContext } from "react";
+
+export const GameContext = createContext();
+
+export function GameProvider({ children, value }) {
+    return <GameContext.Provider value={value}>{children}</GameContext.Provider>
```

Comment:
all state should be placed inside of `GameProvider` and its only prop should be `children`